### ISSUE TITLE
Implement GET /referrals endpoint with mocks

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#98258]
DOES NOT NEED TO BE DONE, DUPLICATE

This is to add the service level referrals (multiple) endpoint (not the controller level). This should connect to the MAP service that pulls HSRM data via the token service, so the token service for MAP will need to be complete before this can be complete.

If you want to split off token integration, you can into a new ticket. It will affect the points on this ticket

**Comment by loripusey** [comment]
@lee-delarm6 add detail

**Comment by lee-delarm6** [comment]
Related issue:

https://github.com/department-of-veterans-affairs/va.gov-team/issues/98259
